keyboards/dztech/dz60/keymaps/mechmerlin/readme.md

```diff
@@ -45,7 +45,7 @@ This is the control layer also known as layer 2. It utilizes the following:
 ---
 ### RESET
 
-As long `BOOTMAGIC_ENABLE` is set to `yes` in `rules.mk`, the DZ60 can be put into bootloader mode by holding the `Escape` key while plugging in. However sometmes this doesn't work or is troublesome to do, might as well use the `RESET` keycode to accomplish this. 
+As long `BOOTMAGIC_ENABLE` is set to `yes` in `rules.mk`, the DZ60 can be put into bootloader mode by holding the `space` key and the `b` key while plugging in. However sometmes this doesn't work or is troublesome to do, might as well use the `RESET` keycode to accomplish this. 
```

Comment:
This change is actually incorrect if the board has a recent enough version of QMK firmware (see [the timeline](https://docs.qmk.fm/#/ChangeLog/20211127?id=bootmagic-full-deprecation-schedule-complete) — since 2021 May 29 setting `BOOTMAGIC_ENABLE = yes` actually enables Bootmagic Lite, and since 2021 Nov 27 all remnants of the “full bootmagic” code were removed completely). A board that was flashed a long time ago might still have a firmware version that requires Space+B, but since this is a personal keymap anyway, you may just leave this alone.